src/workerd/io/worker.h

```diff
@@ -192,6 +163,7 @@ class Worker: public kj::AtomicRefcounted {
 
   class InspectorClient;
   class AsyncWaiter;
+  friend constexpr bool _kj_internal_isPolymorphic(AsyncWaiter*);
```

Comment:
Unfortunate that kj leaks this implementation detail in order to have smart pointers to private forward declared classes...